src/Type/Type.php

```diff
@@ -5,6 +5,10 @@
 interface Type
 {
 
+	const RESULT_YES = 0;
```

Comment:
I'd create a special Result object, but I don't require it now, this PR is mergeable as it is.